examples/StandardRTLSTag_TWR/StandardRTLSTag_TWR.ino

```diff
@@ -213,20 +215,25 @@ void loop() {
         size_t recv_len = DW1000Ng::getReceivedDataLength();
         byte recv_data[recv_len];
         DW1000Ng::getReceivedData(recv_data, recv_len);
-        
+
         /* RTLS standard message */
         if(recv_data[9] == ACTIVITY_CONTROL) {
             if (recv_data[10] == RANGING_CONTINUE) {
                 /* Received Response to poll */
                 timePollSent = DW1000Ng::getTransmitTimestamp();
                 timePollAckReceived = DW1000Ng::getReceiveTimestamp();
                 transmitFinalMessage();
+                String tempString= "Receiving messages from:" ; tempString += (char)anchor_address[0] + (char)anchor_address[1];
+                tempString +=" and send it back.";
+                Serial.println(tempString);
```

Comment:
This print statement blocks interrupts when the device could still be receiving messages.